README.md

```diff
@@ -15,29 +15,33 @@ Mode (choose one):
 Package Options:
   --recursive, -r  Execute the same script on all of its dependencies, too                    [boolean]
   --package, -p    Run only for packages matching this glob. Can be used multiple times.        [array]
+  --changedSince   Runs commands in packages that have changed since the provided source
+                   control branch.                                                             [string]
 
 Misc Options:
-  --fast-exit, e        If at least one script exits with code > 0, abort                     [boolean]
-  --collect-logs, l     Collect per-package output and print it at the end of each script     [boolean]
-  --no-prefix           Don't prefix output                                                   [boolean]
```

Comment:
no-prefix is not removed, its a way to specify a "false" version for `--prefix`

Reply:
I couldn't find reference or usage anywhere in the code. Can you show me where it is used?

Reply:
this is a yargs feature when no-xxx is the same as --xxx=false

Reply:
I've renamed the flag to be `no-prefix` because the documentation (when running `wsrun --help`) would be wrong. 

The previous implementation relied on users knowing the `--no-prefix` yargs trick. The `no-prefix` line was misleading because it's written as an option but you can't add documentation for it so it's just when running `wsrun --help`.

Reply:
i've changed the documentation to include both `--prefix` and `--no-prefix` with explanations on their meaning.